common/views/components/IIIFPresentationPreview/README.md

```diff
@@ -0,0 +1,3 @@
+## Purpose
+
+To display a preview of the content described by a IIIF presentation manifest.
```

Comment:
@Heesoomoon isn't this something to do with relevance?
This sentence is a nice, simple description of what the component is, but what is its purpose?

"To allow researches to make a quick assessment of whether a work is relevant to the research"...?